tests/steps/test_steps.py

```diff
@@ -78,21 +78,11 @@ def test_should_return_error_if_stage_not_defined(self):
             Steps(logger=Logger.manager.getLogger('logger'), properties=properties)
         assert "('invalid' was unexpected)" in excinfo.value.message
 
-    def test_should_succeed_if_executor_is_known(self):
-        project = test_data.get_project_with_stages({'build': 'Echo Build'})
-        result = self.executor.execute(stage=Stage.BUILD, project=project)
-        assert result.output.success
-        assert result.output.message == 'Built test'
-        assert result.output.produced_artifact is None
-
-    def test_should_succeed_if_executor_is_not_known(self):
-        project = test_data.get_project_with_stages({'build': 'Unknown Build'})
-        result = self.executor.execute(stage=Stage.BUILD, project=project)
-        assert not result.output.success
-        assert result.output.message == "Executor 'Unknown Build' for 'build' not known or registered"
-
     def test_should_succeed_if_stage_is_not_known(self):
         project = test_data.get_project_with_stages({'test': 'Some Test'})
         result = self.executor.execute(stage=Stage.BUILD, project=project)
         assert not result.output.success
         assert result.output.message == "Stage 'build' not defined on project 'test'"
+
+    def test_find_all_steps(self):
+        assert list(Step.get_subclasses()) != ""
```

Comment:
We should either assert that the list contains all of the subclasses we have (but that would mean we need to add every new one as well to this test)
Or at least we should assert that the list is not empty